### DISCUSSION TITLE
how to highlight a poling obtained form a geojson

### DISCUSSION THREAD

**Discussion by hugosoli** [heremaps/harp.gl#2322] [Q&A] [answered]
I have my harp mapo working nice. I loaded a geojson with polignons of the cities of a country. I need to trigger an event when somebody click inside that region. The closets example is the "object-picking.ts" hoever in that case the objects belongs to the harp map not the geojason layer. I can not figure out how to transfer that situations. Any help appreciated.
Cheers

**Comment by nzjony** [comment, marked as answer]
Have you seen: https://www.harp.gl/docs/master/examples/#datasource_geojson_styling_game.html ? 

That sounds like it could be similar to what you want?

**Reply by hugosoli** [reply]
This is exactly what I was looking for, Thank you very much
